src/bindings.h

```diff
@@ -31,6 +31,7 @@
 #define LXCFS_NUMSTRLEN64 21
 
 enum lxcfs_virt_t {
+	LXC_TYPE_START,
```

Comment:
I'm not sure that it's safe to change the start value of this enum. Cause we support live lxc library reload after this change `LXC_TYPE_CGDIR` becomes `1` instead of `0`. These values getting placed in the memory and kept between library reloads. So, let's not change this enum declaration at all. You can just place a comment inside enum definition which attracts developer attention to the macros `LXCFS_TYPE_CGROUP`, `LXCFS_TYPE_PROC`, `LXCFS_TYPE_SYS`.

Reply:
I agree here, totally didn't think of the case of reloading.

Generally we restart the service and rebind the mounts in the containers - there is a neat way to do this without restarting the containers, I'm still developing the software but am happy to share it later down the line.

Reply:
> I agree here, totally didn't think of the case of reloading.
> 
> Generally we restart the service and rebind the mounts in the containers - there is a neat way to do this without restarting the containers, I'm still developing the software but am happy to share it later down the line.

great! I'm working on a way to recover the lxcfs from crashes another way. With preserving fuse connection between userspace and kernel. It's also not ready yet :)

Reply:
> I'm not sure that it's safe to change the start value of this enum. Cause we support live lxc library reload after this change `LXC_TYPE_CGDIR` becomes `1` instead of `0`. These values getting placed in the memory and kept between library reloads. So, let's not change this enum declaration at all.

I think we should explictly number it though. That's what I do nowadays for all enums I add becuase it eliminates a whole bunch of subtle bugs if you force the numbering.